### PR TITLE
Add field character limits

### PR DESCRIPTION
**Pull request by tm-kn**
You need to add `{{ form.media }}` to molo-gem templates for survey pages, e.g.
https://github.com/torchbox/molo-gem/commit/da638c937c58db42fceef117ba58ce1bfa7196b1

You also need to have your fields inside `<div class="input-group">` which is the case for Springster templates, but not any other ones.

### CODE REVIEW

File: molo/surveys/templates/surveys/molo_survey_page.html [hunk 1]
```diff
@@ -14,6 +14,7 @@ <h4 class="surveys__intro">{{ page.intro }}</h4>
     {% if form %}
       <form class="surveys__form" action="{% pageurl self %}{% if self.multi_step or self.has_skip_logic %}?p={{ fields_step.number|add:"1" }}{% endif %}" method="post">
         {% csrf_token %}
+        {{ form.media }}
```
Comment (nathanbegbie) [inline molo/surveys/templates/surveys/molo_survey_page.html]: @tm-kn does this evaluate to an import script tag or does it drop the JS straight into the HTML?

Comment (tm-kn) [reply molo/surveys/templates/surveys/molo_survey_page.html]: This evaluates to a <script> and <link> tags. https://docs.djangoproject.com/en/1.9/topics/forms/media/#extend

Comment (nathanbegbie) [reply molo/surveys/templates/surveys/molo_survey_page.html]: @projectmushroom If we just wrap this in our tags to check whether it's freebasics and whether the phone fails - will this work?

Would you rather we added the JS inline?

Comment (projectmushroom) [reply molo/surveys/templates/surveys/molo_survey_page.html]: @nathanbegbie wrapping it in our FB check would be fine 👍 does the form need js to submit? the form would then be useless on freebasics?

Comment (nathanbegbie) [reply molo/surveys/templates/surveys/molo_survey_page.html]: @projectmushroom No - the function of the JS is simply to count down the number of characters remaining while a user types their answer

**Comment by nathanbegbie** [comment]
@tm-kn I'm going to go ahead and merge this code - then in the `molo-gem` update, you will simply need to wrap `{{ form.media }}` with `{% if not is_via_freebasics %} ... {% endif %}`

**Comment by tm-kn** [comment]
@nathanbegbie There's no static character count displayed for single-line text. The `span` label is added by the JS code. We might want to add the static label before we merge?

**Comment by tm-kn** [comment]
@nathanbegbie Added the JS-less count for single-line text fields.

**Comment by nathanbegbie** [comment]
:+1:

**Pull request merged**